### DISCUSSION TITLE
Does waha core session end automatically after some time?

### DISCUSSION THREAD

**Discussion by subramaniasanthosh** [devlikeapro/waha#378] [Q&A]
Im running waha core instance on azure and Im able to connect to my number properly but the when I check in a few days the session is automatically stopped. How to run the session without being stopped automatically?

**Comment by devlikepro** [comment]
It should work for MONTHS if everything goes good (no server restart, no whatsapp interventions, no chrome issues).
Your phone with native app must be online at least every 14 days, otherwise after scanning qr and not seeing the phone online it'll logout you

Please run a session with `debug: true` flag and attach logs right after the session moved to STOPPED status (you can send an email to yourself when it happens) so we can see what is the issue in your case.

https://waha.devlike.pro/docs/how-to/sessions/#enable-debug

Also a additional information would help:
1. What version do you use (make sure to use the latest image)?
2. What OS do you run it on?
3. Is there any errors in logs right before it happens? (see above note for debug)

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
